CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.1.10 (July 9, 2021)
```

Comment:
It probably makes sense to create a new minor version, not a new patch version.

Reply:
done